model.py

```diff
@@ -430,12 +431,11 @@ def forward(self, audio):
         mix_stft = mix_stft.permute(0, 3, 2, 1, 4)
 
         # create an additional target if we need to build a residual
-        targets = self.targets.keys()
         if self.residual is not None:
-            targets += [self.residual]
+            # we add an additional target
```

Comment:
this additional target will be created by the `filtering.wiener`, that will be outputing another target.